pages/common/jwt.md

```diff
@@ -0,0 +1,22 @@
+# jwt
+
+> `jwt-cli` is a command line tool to help you work with JSON Web Tokens (JWTs).
+> A super fast CLI tool to decode and encode JWTs built in Rust.
```

Comment:
I think this line is redundant. We can remove this.

Reply:
Also, we tend to omit adjectives ("complete", "open source", "super fast") in the main description. Stick to describing what it is and what it does :)